### PR TITLE
OS-784 Bug: fix delegatee wrong deletion

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

Task ID: [OS-784](https://aragonassociation.atlassian.net/browse/OS-784)

For QA: https://app.satsuma.xyz/subgraphs/750/versions/5531
query:
```
{
  tokenVotingMembers(where: {plugin: "0x72b1d7ce9748adb10ea7dbd5f9e0456995b02e05"} ) {
    address
    balance
    votingPower
      delegatee {
        address
      }
      delegators {
        address
        balance  
      }
    } 
}
```
## Type of change

See the framework lifecylce in `packages/contracts/docs/framework-lifecycle` to decide what kind of change this pull request is.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [ ] I have updated the `UPDATE_CHECKLIST` file in the root folder.

[OS-784]: https://aragonassociation.atlassian.net/browse/OS-784?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/subgraph/tests/helpers/method-classes.ts [hunk 1]
```diff
@@ -789,10 +790,18 @@ class TokenVotingMemberMethods extends TokenVotingMember {
...
+  ): void {
+    delegatesCall(tokenContractAddress, account, returns);
+  }
+
   createEvent_DelegateChanged(
     delegator: string = this.address.toHexString(),
-    fromDelegate: string = ADDRESS_ONE,
-    toDelegate: string = ADDRESS_ONE,
+    fromDelegate: string = this.address.toHexString(),
+    toDelegate: string = this.address.toHexString(),
```
Comment (Michael-A-Heuer) [inline packages/subgraph/tests/helpers/method-classes.ts:804]: Why was this changed?

File: packages/subgraph/tests/token/governance-erc20.test.ts [hunk 2]
```diff
@@ -239,6 +240,9 @@ describe('Governance ERC20', () => {
   });
 
   describe('handleDelegateChanged', () => {
+    // beforeEach(() => {
+    //   delegatesCall(DAO_TOKEN_ADDRESS, ADDRESS_ONE, ADDRESS_ONE);
+    // });
```
Comment (Michael-A-Heuer) [inline packages/subgraph/tests/token/governance-erc20.test.ts:245]: Please remove

File: packages/subgraph/tests/token/governance-erc20.test.ts [hunk 3]
```diff
@@ -300,20 +304,75 @@ describe('Governance ERC20', () => {
...
+      let pluginAddress = ADDRESS_SIX;
+      let memberOne = new ExtendedTokenVotingMember().withDefaultValues(
+        memberOneAddress,
+        pluginAddress
+      );
+      let memberTwo = new ExtendedTokenVotingMember().withDefaultValues(
+        memberTwoAddress,
+        pluginAddress
+      );
+      /* member one has 100s token delegated to member two*/
```
Comment (Michael-A-Heuer) [inline packages/subgraph/tests/token/governance-erc20.test.ts:341]:
```suggestion
      /* member one has 100 tokens delegated to member two*/
```

**Review by Michael-A-Heuer — CHANGES_REQUESTED** [review]
Minor comment. LGTM otherwise

File: packages/subgraph/tests/token/governance-erc20.test.ts (hunk 3 above)
Comment (Rekard0) [reply packages/subgraph/tests/token/governance-erc20.test.ts]: addressed [e3b807e](https://github.com/aragon/osx/pull/488/commits/e3b807e09d8714ca2bb8110aa3fe9d323d454152)

File: packages/subgraph/tests/token/governance-erc20.test.ts (hunk 2 above)
Comment (Rekard0) [reply packages/subgraph/tests/token/governance-erc20.test.ts]: addressed at [e3b807e](https://github.com/aragon/osx/pull/488/commits/e3b807e09d8714ca2bb8110aa3fe9d323d454152)

**Review by mathewmeconry — APPROVED** [review]

**Review by Michael-A-Heuer — APPROVED** [review]

**Pull request merged**